react-validated-input-component/my-app/src/ValidatedInput.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import { FcCheckmark } from 'react-icons/fc';
+import { FcCancel } from 'react-icons/fc';
+
+export default function ValidatedInput({ underText, symbol }) {
```

Comment:
You pass in `underText` as a prop, but you change it later in the code. It should instead just be a local variable instead of a prop. The same goes for `symbol`.

The best way to think of props is that they are `const`. If you modify them then they probably shouldn't be a prop, but should instead of a local variable or state.